backend/model/Record.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Schema, model, connect, Types } from 'mongoose';
+
+export interface IRecord extends Document {
+    attendanceDate: string;
+    numberOfCheats: Date;
+    evidence: Types.Array<string>;
+    userId: Types.ObjectId;
+    testRoomId: Types.ObjectId;
+  }
+
+  const RecordSchema: Schema = new Schema({
+    attendanceDate: { type: String, required: true, unique: true },
+    numberOfCheats: { type: Date, required: true},
```

Comment:
```suggestion
    numberOfCheats: { type: Number, required: true, default: 0},
```
`numberOfCheats` should start at 0 when document created